app/views/hearings/show.html.haml

```diff
@@ -0,0 +1,75 @@
+= govuk_page_title(@hearing_day.strftime('%d/%m/%Y'), t('generic.hearing'))
+
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = 'Hearing'
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Hearing type
+      %td.govuk-table__cell
+        = @hearing.hearing_type
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Court
+      %td.govuk-table__cell
+        = @hearing.court_name || 'not set'
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Time listed
+      %td.govuk-table__cell
+        = @hearing_day&.strftime('%H:%M')
+
+%br
+.govuk-heading-m
+  = 'Attendees'
+
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = 'Attendees'
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Defendants
+      %td.govuk-table__cell
+        = @hearing.defendant_names.join(', ')
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Defence advocates
+      %td.govuk-table__cell
+        = @hearing.defence_advocate_names&.join(', ') || 'not set'
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Prosecution advocates
+      %td.govuk-table__cell
+        = @hearing.prosecution_advocate_names&.join(', ') || 'not set'
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Judges
+      %td.govuk-table__cell
+        = @hearing.judge_names.join(', ')
+
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = 'Hearing events'
+  %thead.govuk-table__head
+    %tr.govuk-table__row
+      %th.govuk-table__header{ scope: 'col' }= 'Time'
+      %th.govuk-table__header{ scope: 'col' }= 'Event'
+  %tbody.govuk-table__body
+    - @hearing.hearing_events.sort {|a,b| a.occurred_at <=> b.occurred_at }.each do |event|
+      %tr.govuk-table__row
+        %td.govuk-table__cell
+          = event.occurred_at.to_datetime.strftime('%H:%M')
+        %td.govuk-table__cell
+          = event.description
+
+%br
+%br
+= govuk_detail('View raw data') do
+  %h3='Hearings'
+  %pre
+    = JSON.pretty_generate(@hearing.attributes)
+  %h3='Hearing events'
+  %pre
+    = JSON.pretty_generate(@hearing.hearing_events.map(&:attributes))
```

Comment:
Don't forget you have this here, it would be ideal if we could show this based on a user role

@jsugarman

Reply:
have added conditional logic for now based an env var